app/viewmodels/cya/CYAYourDetailsSummaryListHelper.scala

```diff
@@ -55,46 +56,30 @@ trait CYAYourDetailsSummaryListHelper {
 
     val contactDetailsSummaryListRow: Seq[SummaryListRow] = request.userAnswers.get(DeclarantContactDetailsPage) match {
       case Some(details) =>
+        val contactDetailsVaalue = {
+          details.fullName + "<br/>" +
+            details.email + "<br/>" +
+            details.phoneNumber
+        }
         Seq(
           SummaryListRow(
             key = Key(
               content = Text(messages("cya.contactDetails")),
-              classes = "govuk-!-width-two-thirds govuk-!-padding-bottom-0"
+              classes = "govuk-!-width-two-thirds govuk-!-padding-top-0"
```

Comment:
because you only have class govuk-!-padding-top-0 on 2 of the 3 fields in the row, it has no effect. I suspect you don't need it unless you want to close the distance up on the heading? Happy to review these with you as there are a few instances where we have applied classes in only 2 places so they are ignored!